pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
```

Comment:
This file is not needed, there is already a pryproject.toml under the docs folder.

Please delete this file and keep "docs/pyproject.toml"